zerver/lib/markdown/__init__.py

```diff
@@ -2282,6 +2358,7 @@ def build_treeprocessors(self) -> markdown.util.Registry:
         )
         if settings.CAMO_URI:
             treeprocessors.register(InlineImageProcessor(self), "rewrite_images_proxy", 10)
+            treeprocessors.register(InlineVideoProcessor(self), "rewrite_videos_proxy", 10)
```

Comment:
Not sure if this works since I don't have CAMO setup locally.

Reply:
Works as per tests.

Reply:
Tests won't verify that Go-camo is capable of proxying the preview correctly.  Can you make sure to end-to-end test that?